dwave/preprocessing/presolve/cypresolve.pyx

```diff
@@ -196,11 +197,17 @@ cdef class cyPresolver:
 
         return changes
 
-    cpdef bint presolve(self) except*:
+    cpdef bint presolve(self, double time_limit = float("inf")) except*:
```

Comment:
Personal preference but I like to put the unit in the arg name so something like `time_limit_seconds`

Reply:
Hmm, I had chosen it to be consistent with the CQM solver but... you're right, why not.